config/pipelines/tasks/meteor_oc-actions.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: oc-actions
```

Comment:
If there is no `url-to-results` in the pipeline, then this task is not needed either